Library/Formula/curl.rb

```diff
@@ -52,9 +53,9 @@ def install
       args << "--with-ssl=#{Formula["libressl"].opt_prefix}"
       args << "--with-ca-bundle=#{etc}/libressl/cert.pem"
     else
-      ENV.prepend_path "PKG_CONFIG_PATH", "#{Formula["openssl"].opt_lib}/pkgconfig"
-      args << "--with-ssl=#{Formula["openssl"].opt_prefix}"
-      args << "--with-ca-bundle=#{etc}/openssl/cert.pem"
+      ENV.prepend_path "PKG_CONFIG_PATH", "#{Formula["openssl3"].opt_lib}/pkgconfig"
+      args << "--with-ssl=#{Formula["openssl3"].opt_prefix}"
+      args << "--with-ca-bundle=#{etc}/openssl@3/cert.pem"
```

Comment:
Wait a sec, I think this is wrong.

Reply:
Do you mean the path set `--with-ca-bundle`? because the [openssl3 formula sets it to that](https://github.com/mistydemeo/tigerbrew/blob/ee4014d1451a1ae45811e433eb91a27877b63c4d/Library/Formula/openssl3.rb#L63) I thought it was best to keep things separate from the openssl 1.1.1 formula as I suspect things will live side by side for some time.